CLR/Libraries/SPOT_Graphics/spot_graphics_native_Microsoft_SPOT_Bitmap.cpp

```diff
@@ -766,7 +767,7 @@ HRESULT Library_spot_graphics_native_Microsoft_SPOT_Bitmap::TileImage___VOID__I4
                 dst.right  = dst.left + w - 1;
                 dst.bottom = dst.top + h - 1;
 
-     	        bitmap->DrawImage( dst, *bitmapSrc, src, opacity );
+                bitmap->DrawImage( dst, *bitmapSrc, src, opacity );

```

Comment:
No changes here outside line endings.  The diff appears to be confused by the mixed line endings being fixed.  Same for below.